### PR TITLE
Add Sandbag to editor

### PR DESCRIPTION
**Pull request by buddytex**
As per #2633 

Make Sandbag available for placement in editor (Under Objects)
Also renames ACE_SandbagObject to ACE_Sandbag

### CODE REVIEW

**Comment by jonpas** [comment]
That breaks backwards compatibility, I wouldn't change the classname.

**Comment by buddytex** [comment]
@jonpas I doubt that is much of an issue but @commy2 asked for it.

**Comment by jokoho48** [comment]
i think you better only change the visibilty or you add a backward compatible

File: addons/sandbag/config.cpp [hunk 1]
```diff
@@ -6,7 +6,7 @@ class CfgPatches {
         weapons[] = {"ACE_Sandbag", "ACE_Sandbag_empty"};
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"ace_interaction"};
-        author[] = {"Rocko", "Ruthberg"};
+        author[] = {"Rocko", "Ruthberg", "Buddytex"};
```
Comment (jokoho48) [inline addons/sandbag/config.cpp]: i think thats a little bit to much :S

**Comment by buddytex** [comment]
@jokoho48 There only the visibility change, I'm not entirely sure how to ensure backwards compatibility with a class in ArmA.

File: addons/sandbag/CfgVehicles.hpp [hunk 2]
```diff
@@ -46,10 +46,11 @@ class CfgVehicles {
     class ACE_SandbagObject: thingX {
         author = ECSTRING(common,ACETeam);
         XEH_ENABLED;
-        scope = 1;
+        scope = 2;
         side = -1;
         model = PATHTOF(data\ace_sandbag_build.p3d);
         displayName = CSTRING(sandbag_displayName);
+		vehicleClass = "Objects";
```
Comment (jonpas) [inline addons/sandbag/CfgVehicles.hpp]: Tabs instead of spaces.

**Comment by commy2** [comment]
Let's please change the classname before adding it to the editor. ACE_Sandbag__Object__ ...

File: addons/sandbag/CfgVehicles.hpp [hunk 3]
```diff
@@ -46,10 +46,11 @@ class CfgVehicles {
     class ACE_SandbagObject: thingX {
         author = ECSTRING(common,ACETeam);
         XEH_ENABLED;
-        scope = 1;
+        scope = 2;
         side = -1;
         model = PATHTOF(data\ace_sandbag_build.p3d);
         displayName = CSTRING(sandbag_displayName);
+        vehicleClass = "Objects";
```
Comment (jonpas) [inline addons/sandbag/CfgVehicles.hpp]: This should probably be `"ACE_Logistics_Items"`;

**Comment by miklide** [comment]
Am I missing something? [img](https://i.imgur.com/5wlERXJ.jpg) [img2](https://i.imgur.com/SZPXzPM.jpg)

**Pull request merged**